test/interface_test.py

```diff
@@ -67,6 +70,19 @@ def test_interface_run_with_run_failure(self):
 
         self.assertFalse(self._run_interface())
 
+    def test_just_run_main_task_cls(self):
+        class MyTestTask(luigi.Task):
+            pass
+
+        class MyOtherTestTask(luigi.Task):
+            my_param = luigi.Parameter()
+
+        with patch.object(sys, 'argv', ['my_module.py', '--no-lock', '--local-scheduler']):
+            luigi.run(main_task_cls=MyTestTask)
```

Comment:
Well yes, but the whole purpose is to test the behaviour when you don't pass that argument. As the commit message says. :)